cwms-data-api/src/test/java/cwms/cda/api/ControllersTest.java

```diff
@@ -245,4 +246,67 @@ void testDoubleQueryParamNull() {
         assertNull(intervalForward);
 
     }
+
+    @Test
+    void testDeleteMethod(){
+        JooqDao.DeleteMethod deleteMethod = Controllers.getDeleteMethod(null);
+        assertNull(deleteMethod);
+
+        try {
```

Comment:
Use assertThrows
`assertThrows(IllegalArgumentException.class, () -> { deleteMethod = ...}, "Expected exception not thrown")`

This also makes it more clear in the build run output when the wrong exception gets thrown.